platform/o.n.swing.plaf/src/org/netbeans/swing/plaf/AllLFCustoms.java

```diff
@@ -154,6 +154,7 @@ public static void initCustomFontSize (int uiFontSize) {
         switchFont("windowTitleFont", fontTranslation, uiFontSize, nbDialogBold); // NOI18N
         switchFont("Spinner.font", fontTranslation, uiFontSize, nbDialogPlain); // NOI18N
         switchFont("FormattedTextField.font", fontTranslation, uiFontSize, nbDialogPlain); // NOI18N
+        switchFont("Slider.font", fontTranslation, uiFontSize, nbDialogPlain); // NOI18N
```

Comment:
:+1: Should add `// NOI18N`s to strings of the other files that are not expected translation as well.

Reply:
Will do.